assets/eip-7432/ERC7432.sol

```diff
@@ -86,39 +132,9 @@ contract ERC7432 is IERC7432 {
         return interfaceId == type(IERC7432).interfaceId;
     }
 
-    function grantRoleFrom(
-        bytes32 _role,
-        address _tokenAddress,
-        uint256 _tokenId,
-        address _grantor,
-        address _grantee,
-        uint64 _expirationDate,
-        bytes calldata _data
-    ) external override validExpirationDate(_expirationDate) {
-        require(tokenApprovals[_grantor][_tokenAddress][_tokenId][msg.sender] || operatorApprovals[_grantor][_tokenAddress][msg.sender], "RolesRegistry: sender must be approved");
-
-        roleAssignments[_grantor][_grantee][_tokenAddress][_tokenId][_role] = RoleData(_expirationDate, _data);
-        lastRoleAssignment[_grantor][_tokenAddress][_tokenId][_role] = _grantee;
-        emit RoleGranted(_grantor, _role, _tokenAddress, _tokenId, _grantee, _expirationDate, _data); // TODO: We should change event to receive grantor as parameter
-    }
-
-    function revokeRoleFrom(
-        bytes32 _role,
-        address _tokenAddress,
-        uint256 _tokenId,
-        address _grantor,
-        address _grantee
-    ) external override {
-        require(tokenApprovals[_grantor][_tokenAddress][_tokenId][msg.sender] || operatorApprovals[_grantor][_tokenAddress][msg.sender], "RolesRegistry: sender must be approved");
-        
-        delete roleAssignments[_grantor][_grantee][_tokenAddress][_tokenId][_role];
-        delete lastRoleAssignment[_grantor][_tokenAddress][_tokenId][_role];
-        emit RoleRevoked(_grantor, _role, _tokenAddress, _tokenId, _grantee); // TODO: We should change event to receive grantor as parameter
-    }
-
     function setRoleApprovalForAll(
-        address _operator,
         address _tokenAddress,
+        address _operator,
         bool _isApproved
     ) external override {
         operatorApprovals[msg.sender][_tokenAddress][_operator] = _isApproved;
```

Comment:
mapping `operatorApprovals` was not declared. I would rename it to `tokenApprovals`